src/db.ts

```diff
@@ -22,6 +22,21 @@ export const getCouchSetup = (config: Config): CouchDbSetup => {
       partitioned: true
     }
   }
+  const conflictsDesign: JsDesignDocument = {
+    language: 'javascript',
+    views: {
+      conflictRevs: {
+        map: stringifyCode(function (doc) {
+          if (doc._conflicts != null) {
+            emit(null, [doc._rev].concat(doc._conflicts))
+          }
+        })
+      }
+    },
+    options: {
+      partitioned: true
```

Comment:
I think you are going to need another flag here to tell Couch to add `_conflicts`, but I don't remember the flag name.

Reply:
I think the field is present for the doc in the view function regardless of any flag: https://docs.couchdb.org/en/latest/replication/conflicts.html#view-map-functions

Reply:
Ah, I see, it was the local sequence number you have to ask for specially.